pkg/virt-controller/services/template.go

```diff
@@ -527,7 +534,10 @@ func (t *templateService) renderLaunchManifest(vmi *v1.VirtualMachineInstance, t
 			})
 		}
 		if volume.Ephemeral != nil {
-			volumeMounts = append(volumeMounts, volumeMount)
+			claimName := volume.Ephemeral.PersistentVolumeClaim.ClaimName
+			if err := t.addPVCToLaunchManifest(volume, claimName, namespace, &volumeMounts, &volumeDevices); err != nil {
+				return nil, err
+			}
```

Comment:
this looks like it could use a unit test case in template_test.go. probably under the `"with blockdevice mode pvc source"` context